src/Collection/Iterator/ZipIterator.php

```diff
@@ -90,7 +89,7 @@ public function __construct(array $sets, ?callable $callable = null)
      *
      * @return array
      */
-    #[ReturnTypeWillChange]
+    #[\ReturnTypeWillChange]
```

Comment:
We could enable the `allowFullyQualifiedGlobalClasses` option for this sniff https://github.com/slevomat/coding-standard#slevomatcodingstandardnamespacesreferenceusednamesonly-.

Reply:
Would be nice if we could allow only some classes instead of all `\DateTime` names, but I don't think that's possible.

Wish attributes were recognized better. We could move the CS+SA step to php 8.

Reply:
Yeah, I just reverted to the `use` statement for now.